android/app/build.gradle

```diff
@@ -105,6 +105,7 @@ android {
         applicationId "com.nativetemplate"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
+        missingDimensionStrategy 'react-native-camera', 'general'
```

Comment:
It might make sense to be consistent with the quotes. I see double and single quotes in this file.

Reply:
done!